### DISCUSSION TITLE
Formatting of property files

### DISCUSSION THREAD

**Discussion by TrainmasterHD** [WeblateOrg/weblate#11642] [Q&A] [answered]
Hi,
we are using property files for storing our translations. But we historically have used another formatting than weblate is formatting the files. For example a comparison between our and weblate's formatting:

Our: 
```
key = This is a localized string\n\
This is a new line
```

Weblate: 
```
key=This is a localized string\nThis is a new line
```

Is it possible to change this behavior or not? At least it would be great because the formatting we are using is a bit better to view. And also we need to adjust all files to match the weblate formatting and to not have all files to have pending changes

**Comment by nijel** [comment, marked as answer]
I don't think that translate-toolkit (library Weblate uses to handle translation formats) supports emitting properties in this way.

**Reply by TrainmasterHD** [reply]
So, you think it will not be possible to format it like the original format?

**Reply by nijel** [reply]
No, translate-toolkit IMHO doesn't emit line continuations at all.

**Reply by TrainmasterHD** [reply]
Okay, then I think we have to live with the new formatting of the files. In the old localization tool it worked with the formatting. Thank you for your answer
